sklearn/utils/sparsefuncs_fast.pyx

```diff
@@ -48,6 +50,63 @@ def csr_row_norms(X):
 @cython.boundscheck(False)
 @cython.wraparound(False)
 @cython.cdivision(True)
```

Comment:
I think you can put that those once and for all at the top of the `.pyx` file as:

```cython
# cython: cdivision=True
# cython: boundscheck=False
# cython: wraparound=False
```